### DISCUSSION TITLE
Can’t use entries with “Master password re-prompt” enabled

### DISCUSSION THREAD

**Discussion by jiams95** [dani-garcia/vaultwarden#3904] [Q&A] [answered]
### Subject of the issue
When I try to connect to a website ,for which I activated the "Master password re-prompt", Bitwarden (iOS 17) asks for the master password and then just acts like usually (app is closed and I get back to browser) but never paste the password and never copies the 2FA code. 

### Deployment environment
<details><summary>Show Environment (Generated via diagnostics page)</summary>
 
* Vaultwarden version: v1.29.2-bbd630f1
* Web-vault version: v2023.8.2
* OS/Arch: linux/x86_64
* Running within Docker: true (Base: Debian)
* Environment settings overridden: true
* Uses a reverse proxy: false
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.41.2
* Clients used: issue in iOS, other clients used browser extension for Google Chrome and Windows app
* Reverse proxy and version: 
* Other relevant information: 
</details>

### Config
<details><summary>Show Running Config (Generated via diagnostics page)</summary>
 
**Environment settings which are overridden:** SIGNUPS_ALLOWED

```json
{
  "_duo_akey": null,
  "_enable_duo": false,
  "_enable_email_2fa": true,
  "_enable_smtp": true,
  "_enable_yubico": false,
  "_icon_service_csp": "",
  "_icon_service_url": "",
  "_ip_header_enabled": true,
  "_smtp_img_src": "cid:",
  "admin_ratelimit_max_burst": 3,
  "admin_ratelimit_seconds": 300,
  "admin_session_lifetime": 20,
  "admin_token": "***",
  "allowed_iframe_ancestors": "",
  "attachments_folder": "data/attachments",
  "auth_request_purge_schedule": "30 * * * * *",
  "authenticator_disable_time_drift": false,
  "data_folder": "data",
  "database_conn_init": "",
  "database_max_conns": 10,
  "database_timeout": 30,
  "database_url": "***************",
  "db_connection_retries": 15,
  "disable_2fa_remember": false,
  "disable_admin_token": false,
  "disable_icon_download": false,
  "domain": "*****://*************",
  "domain_origin": "*****://*************",
  "domain_path": "",
  "domain_set": true,
  "duo_host": null,
  "duo_ikey": null,
  "duo_skey": null,
  "email_attempts_limit": 3,
  "email_expiration_time": 600,
  "email_token_size": 6,
  "emergency_access_allowed": true,
  "emergency_notification_reminder_schedule": "0 3 * * * *",
  "emergency_request_timeout_schedule": "0 7 * * * *",
  "enable_db_wal": true,
  "event_cleanup_schedule": "0 10 0 * * *",
  "events_days_retain": null,
  "extended_logging": false,
  "helo_name": "*************",
  "hibp_api_key": null,
  "icon_blacklist_non_global_ips": true,
  "icon_blacklist_regex": null,
  "icon_cache_folder": "data/icon_cache",
  "icon_cache_negttl": 259200,
  "icon_cache_ttl": 2592000,
  "icon_download_timeout": 10,
  "icon_redirect_code": 302,
  "icon_service": "internal",
  "incomplete_2fa_schedule": "30 * * * * *",
  "incomplete_2fa_time_limit": 3,
  "invitation_expiration_hours": 120,
  "invitation_org_name": "******* Bitwarden",
  "invitations_allowed": true,
  "ip_header": "X-Forwarded-For",
  "job_poll_interval_ms": 30000,
  "log_file": "/data/logs/vaultwarden.txt",
  "log_level": "info",
  "log_timestamp_format": "%d-%m-%Y %H:%M:%S.%3f",
  "login_ratelimit_max_burst": 10,
  "login_ratelimit_seconds": 60,
  "org_attachment_limit": null,
  "org_creation_users": "",
  "org_events_enabled": false,
  "org_groups_enabled": true,
  "password_hints_allowed": true,
  "password_iterations": 100000,
  "push_enabled": false,
  "push_installation_id": "***",
  "push_installation_key": "***",
  "push_relay_uri": "https://push.bitwarden.com",
  "reload_templates": false,
  "require_device_email": true,
  "rsa_key_filename": "data/rsa_key",
  "send_purge_schedule": "0 5 * * * *",
  "sendmail_command": null,
  "sends_allowed": true,
  "sends_folder": "data/sends",
  "show_password_hint": false,
  "signups_allowed": false,
  "signups_domains_whitelist": "**********",
  "signups_verify": true,
  "signups_verify_resend_limit": 6,
  "signups_verify_resend_time": 3600,
  "smtp_accept_invalid_certs": true,
  "smtp_accept_invalid_hostnames": false,
  "smtp_auth_mechanism": "Plain",
  "smtp_debug": false,
  "smtp_embed_images": true,
  "smtp_explicit_tls": null,
  "smtp_from": "**************************",
  "smtp_from_name": "******* - Vaultwarden",
  "smtp_host": "**********",
  "smtp_password": "***",
  "smtp_port": 587,
  "smtp_security": "starttls",
  "smtp_ssl": null,
  "smtp_timeout": 15,
  "smtp_username": "***************",
  "templates_folder": "data/templates",
  "tmp_folder": "data/tmp",
  "trash_auto_delete_days": null,
  "trash_purge_schedule": "0 5 0 * * *",
  "use_sendmail": false,
  "use_syslog": false,
  "user_attachment_limit": null,
  "web_vault_enabled": true,
  "web_vault_folder": "web-vault/",
  "websocket_address": "0.0.0.0",
  "websocket_enabled": true,
  "websocket_port": 3012,
  "yubico_client_id": null,
  "yubico_secret_key": null,
  "yubico_server": null
}
```

* vaultwarden version: v1.29.2-bbd630f1

* Install method: Docker image in Synology NAS

* Clients used: issue in iOS, other clients used browser extension for Google Chrome and Windows app

* Reverse proxy and version: -

* MySQL/MariaDB or PostgreSQL version: SQLite 3.41.2

* Other relevant details: Bitwarden iOS Version: 2023.8.0 (4566)
</details>

### Steps to reproduce
1. Create a password with the "Master password re-prompt" active (and a 2FA, but it is not mandatory to test the issue)
2. Using an iPhone, with Google Chrome and Bitwarden installed, open Google Chrome and go to the website where the password should be entered
3. Enter in the login or password field and click on the Bitwarden icon to select a password
4. Select the password created in the 1st step and type the master password as prompted and confirm
5. The Bitwarden app closes and you will get back to Google Chrome where the login/password is not pasted as it should be
6. If you configured a 2FA, try to paste and see that the 2FA code is not copied in clipboard neither

/!\ Please note that going directly to Bitwarden app and trying to copy the password, it doesn't work as well and trying to see it makes the app to crash.

### Expected behaviour
I expected the login/password is pasted automatically and the 2FA code is copied in clipboard.

### Actual behaviour
Nothing happens after master password is confirmed. 

### Troubleshooting data
Please tell me if you need something specific here. 

Thanks for your help!

**Comment by quexten** [comment]
I believe this is an issue with the Bitwarden mobile app introduced in ~2023.08.x. Vaultwarden is only the backend server so it's unrelated.

**Comment by dfunkt** [comment, marked as answer]
https://github.com/bitwarden/clients/issues/6268#issuecomment-1717124788

**Reply by jiams95** [reply]
Thanks!
